spec/features/cheeting_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+feature "User makes a new cheet" do
+  scenario "when on the homepage" do
```

Comment:
again, When I run rspec all I see it 'when on the homepage' twice. This is a far more complicated test than 'when on the homepage'. context 'on the home page',  'creating a tweet', 'displaying a tweet'.